src/ucp/core/ucp_request.h

```diff
@@ -121,10 +122,11 @@ struct ucp_request {
                 } proxy;
 
                 struct {
-                    uint64_t      remote_address; /* address of the sender's data buffer */
-                    uintptr_t     remote_request; /* pointer to the sender's send request */
-                    uct_rkey_bundle_t rkey_bundle;
-                    ucp_request_t *rreq;    /* receive request on the recv side */
+                    uint64_t           remote_address; /* address of the sender's data buffer */
+                    uintptr_t          remote_request; /* pointer to the sender's send request */
+                    uct_rkey_bundle_t  rkey_bundle;
+                    ucp_request_t     *rreq;           /* receive request on the recv side */
+                    struct ucp_rndv_get_mrail *mrail;  /* multirail info */
```

Comment:
the convention is to prefer ucp_rndv_get_mrail_t

Reply:
there was forward declaration, fixed